Makefile

```diff
@@ -31,8 +31,8 @@ PYENV_PREREQ_HELP = "\033[0;31mIMPORTANT\033[0m: please type \033[0;31mpyenv ini
 VE_MISSING_HELP = "\033[0;31mIMPORTANT\033[0m: Couldn't find $(PWD)/$(VIRTUAL_ENV); have you executed make venv-create?\033[0m\n"
 
 prereq:
-	pyenv install --skip-existing $(PY38)
-	pyenv local $(PY38)
+	pyenv install --skip-existing $(PY312)
```

Comment:
How come we jump to 3.12 instead of 3.9 ?

Reply:
This follows dev environment preparation from Rally's Makefile, where 3.12 is used as well. It aligns both.

Reply:
Ah, I hadnt spotted we had switched there (We used to use 3.8, then when we upgraded to 3.12 we jumped straight to 3.12). I suppose from a testing perspective we still test on 3.9, so thats ok I guess